daml-lf/language/src/main/scala/com/digitalasset/daml/lf/language/PackageInterface.scala

```diff
@@ -418,6 +418,28 @@ private[lf] class PackageInterface(signatures: PartialFunction[PackageId, Packag
   val packageLanguageVersion: PartialFunction[PackageId, LanguageVersion] =
     signatures andThen (_.languageVersion)
 
+  def interfaceImplementations(
+      pkgId: PackageId,
+      acc: Map[TypeConName, Set[TypeConName]] = Map.empty,
```

Comment:
Consider making the function that takes this argument private and add an overload without that argument to reduce your API surface.

Reply:
not applicable anymore